src/plugin.js

```diff
@@ -142,6 +143,10 @@ function zoomNumericalScale(scale, zoom, center, zoomOptions) {
 	var minDelta = newDiff * minPercent;
 	var maxDelta = newDiff * maxPercent;
 
+	if (scale.type === 'logarithmic') {
+		tickOpts.min = tickOpts.min < 0 ? 0 : tickOpts.min;
+	}
+
 	scale.options.ticks.min = rangeMinLimiter(zoomOptions, scale.min + minDelta);
```

Comment:
Hi, thanks for your Pull Request!

I tested it (in the sample `zoom-log.html`) but I can't get it work: the bug is still here. I think that the `tickOpts.min =` assignment is overridden by the next `scale.options.ticks.min =` assignment.

You could write something like:

```js
var newMin = rangeMinLimiter(zoomOptions, scale.min + minDelta);

if (scale.type === 'logarithmic') {
  newMin = Math.max(newMin, 0);
}

scale.options.ticks.min = newMin;
scale.options.ticks.max = rangeMaxLimiter(zoomOptions, scale.max - maxDelta);
```

Reply:
OMG... I forgot the important stuff =(

sorry... now it should be ok

Reply:
No problem. It's actually better. :wink: